src/services/queue.js

```diff
@@ -0,0 +1,10 @@
+import Queue from 'bull';
+
+const REDIS_PORT = process.env.REDIS_PORT || 6379;
+const { REDIS_HOST, REDIS_PASS } = process.env;
```

Comment:
This will likely need to be updated (similar to the s3 configuration) for cloud.gov provided credentials, but that can wait until we set up the infrastructure as long as the setup is all limited to this file.